ssl-config-core/src/main/resources/reference.conf

```diff
@@ -131,60 +128,34 @@ ssl-config {
 
   # Debug configuration
   debug {
-
-    # Turn on all debugging
+    # Enable all debugging
     all = false
 
-    # Turn on ssl debugging
+    # Enable sslengine / socket tracing
     ssl = false
 
-    # Turn certpath debugging on
-    certpath = false
-
-    # Turn ocsp debugging on
-    ocsp = false
-
-    # Enable per-record tracing
-    record = false
-
-    # hex dump of record plaintext, requires record to be true
-    plaintext = false
-
-    # print raw SSL/TLS packets, requires record to be true
-    packet = false
-
-    # Print each handshake message
-    handshake = false
-
-    # Print hex dump of each handshake message, requires handshake to be true
-    data = false
-
-    # Enable verbose handshake message printing, requires handshake to be true
-    verbose = false
-
-    # Print key generation data
-    keygen = false
-
-    # Print session activity
-    session = false
-
-    # Print default SSL initialization
-    defaultctx = false
-
-    # Print SSLContext tracing
+    # Enable SSLContext tracing
     sslctx = false
 
-    # Print session cache tracing
-    sessioncache = false
-
-    # Print key manager tracing
+    # Enable key manager tracing
     keymanager = false
 
-    # Print trust manager tracing
+    # Enable trust manager tracing
     trustmanager = false
 
-    # Turn pluggability debugging on
-    pluggability = false
-
+    // The following settings are deprecated and have no effect in code.
+    certpath = false # DEPRECATED
```

Comment:
It would be good to have a warning logged when these are configured to `true`.

Reply:
Added in https://github.com/lightbend/ssl-config/pull/143/commits/1660915e77c3e0eb0c41b9493fc42586c589808c